### ISSUE TITLE
[quality] ControlPoints is both in pyFAI.control_points and pyFAI.peak_picker

### ISSUE THREAD

**Issue by vallsv** [silx-kit/pyFAI#798]
A very fast check make me think it's the same thing.
@kif could you confirm?
I guess we can remove the one from `peak_picker`? Like that there is no dependancy from `goniometer` to `peak_picker`?

**Comment by kif** [comment]
On Mon, 05 Mar 2018 07:53:17 -0800
Valentin Valls <notifications@github.com> wrote:

> A very fast check make me think it's the same thing.
> @kif could you confirm?
> I guess we can remove the one from `peak_picker`? Like that there is no dependancy from `goniometer` to `peak_picker`?

I moved the ControlPoint to its own file as pesk-picker has Qt stuff in it and I did not want any Qt dependency on Goniometer.

If it still is in peak_picker, it is either a mistake or a "merge-error" at the git level.

Feel free to remove it from pek-picker.

Cheers,

J.
